ci/scripts/github/checks.sh

```diff
@@ -55,9 +60,6 @@ export MORPHEUS_IN_SPHINX_BUILD=1
 rapids-logger "Checking copyright headers"
 python ${MORPHEUS_ROOT}/ci/scripts/copyright.py --verify-apache-v2 --git-diff-commits ${CHANGE_TARGET} ${GIT_COMMIT}
 
-rapids-logger "Running Python style checks"
-${MORPHEUS_ROOT}/ci/scripts/python_checks.sh
-
```

Comment:
Were removing the python checks but are they added in pre-commit?

Reply:
Oh good catch. I'm not intending to add the python checks in this PR.